apps/jobbrowser/src/jobbrowser/apis/stream.py

```diff
@@ -0,0 +1,16 @@
+from django.http import StreamingHttpResponse
+
+from desktop.lib.rest.http_client import HttpClient
+
+CHUNK_SIZE = 16 * 1024
+
+def stream(client: HttpClient, url: str):
+  stream_resp = client.execute("GET", url, stream=True);
+  response = StreamingHttpResponse(_stream_generate(stream_resp), content_type=stream_resp.headers.get('Content-Type'))
+  response['Content-Disposition'] = stream_resp.headers.get('Content-Disposition')
+
+  return response
+
```

Comment:
`stream.py` might not be super explicit naming, maybe move it to desktop/core/src/desktop/lib/rest/ to something similar to `stream_response()` in `streamed_resource.py`? (not perfect but bit clearer)

Reply:
Thanks, done.